src/unix/kqueue.c

```diff
@@ -53,11 +57,12 @@ int uv__io_check_fd(uv_loop_t* loop, int fd) {
   int rc;
 
   rc = 0;
-  EV_SET(&ev, fd, EVFILT_READ, EV_ADD, 0, 0, 0);
+  EV_SET(&ev, fd, EVFILT_READ, EV_ADD | EV_OOBAND, 0, 0, 0);
```

Comment:
I don't think this is needed.  Or is there some device which will work with pollpri but not pollin?